.github/workflows/run_detekt_and_unit_tests.yml

```diff
@@ -38,8 +39,8 @@ jobs:
           name: DetektReportsTemplate
           path: template/build/reports/detekt/
```

Comment:
should we update the path for detekt and `Archive code coverage reports on template` step too?